test/extensions/geoip_providers/maxmind/config_test.cc

```diff
@@ -331,19 +370,29 @@ TEST(MaxmindProviderConfigTest, DifferentProviderInstancesForDifferentProtoConfi
         anon_hosting: "x-anon-hosting"
     city_db_path: %s
     anon_db_path: %s
+    country_mapping_path: %s
   )EOF";
   MaxmindProviderConfig provider_config1;
   MaxmindProviderConfig provider_config2;
   auto city_db_path = genGeoDbFilePath("GeoLite2-City-Test.mmdb");
   auto asn_db_path = genGeoDbFilePath("GeoLite2-ASN-Test.mmdb");
   auto anon_db_path = genGeoDbFilePath("GeoIP2-Anonymous-IP-Test.mmdb");
-  auto processed_provider_config_yaml1 =
-      absl::StrFormat(provider_config_yaml1, city_db_path, asn_db_path, anon_db_path);
+  auto country_mapping_path = genGeoDbFilePath("country_mapping.conf");
+
+  auto processed_provider_config_yaml1 = absl::StrFormat(
+      provider_config_yaml1, city_db_path, asn_db_path, anon_db_path, country_mapping_path);
   auto processed_provider_config_yaml2 =
-      absl::StrFormat(provider_config_yaml2, city_db_path, anon_db_path);
+      absl::StrFormat(provider_config_yaml2, city_db_path, anon_db_path, country_mapping_path);
   TestUtility::loadFromYaml(processed_provider_config_yaml1, provider_config1);
   TestUtility::loadFromYaml(processed_provider_config_yaml2, provider_config2);
   NiceMock<Server::Configuration::MockFactoryContext> context;
+
+  auto api = Api::createApiForTest();
+
+  Stats::IsolatedStoreImpl stats_store_;
+  Stats::ScopeSharedPtr scope_{stats_store_.createScope("")};
```

Comment:
these variables are local to a single test code, the names should not have underscores in the end.